src/redux/dragon/dragonSlice.js

```diff
@@ -0,0 +1,60 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'https://api.spacexdata.com/v4/dragons';
+
+export const getDragon = createAsyncThunk(
+  'books/getDragons',
+  async () => {
+    const res = await axios.get(
+      url,
+    );
+    const dragons = Object.keys(res.data).map((key) => ({
+      mission_id: key,
+      ...res.data[key],
+    }));
+    return dragons;
```

Comment:
- [ ] Storing the entire data fetched from the API in Redux may result in unnecessary overhead and slow down the application. It would be great to only store the information necessary for this application. Kindly map the incoming data to store the required fields suggested in the project requirements. 😎️
You did a very good job filtering the data in the rockets and missions slices, let's do the same thing for dragons. 

![Screenshot from 2023-03-24 17-30-07](https://user-images.githubusercontent.com/17243325/227556365-732776c5-77f9-4cb3-9d85-be85e855f41a.png)

Reply:
Fixed.